gensim/models/word2vec.py

```diff
@@ -1576,15 +1576,17 @@ def accuracy(self, questions, restrict_vocab=30000, most_similar=most_similar, u
         Use `restrict_vocab` to ignore all questions containing a word whose frequency
         is not in the top-N most frequent words (default top 30,000).
 
-        Use `use_lowercase` to convert all words in questions to thier lowercase form before evaluating
-        the accuracy. It's useful when assuming the text preprocessing also uses lowercase.  (default True).
+        Use `case_insensitive` to convert all words in questions and vocab to their lowercase form before evaluating
```

Comment:
It shouldn't make a difference in results, but in the spirit of mimicking `compute-accuracy.c` unless there's a good reason not to, perhaps we should case-normalize to uppercase?